README.md

```diff
@@ -182,6 +182,69 @@ You can intermittently poll whether the operation has finished by using the `Pol
   }
 ```
 
+#### `Azure::Core::Context`
+
+Most Azure SDK Service Client methods accept an optional `Azure::Core::Context` parameter, which is used to enable cancellation of the operation or to
+establish an absolute deadline for the operation.
+
+This is useful when you want to assign a time limit on an operation to ensure that it completes in a "reasonable" timeframe. For instance, the 
+snippet below will cancel a blob client upload after 5 seconds.
+
+<!-- @insert_snippet: CreateBlobContext -->
+```cpp
+      Azure::Core::Context cancelledIn5s = Azure::Core::Context::CreateWithDeadline(
```

Comment:
BTW, do we still have this `CreateWithDeadline()` method, I can't seem to find it.

Reply:
No, we don't. It's a ctor overload, so we should update this sample snippet.